app/src/main/res/layout/activity_movie_details.xml

```diff
@@ -0,0 +1,317 @@
+            app:layout_constraintEnd_toEndOf="@id/right_guidline"
+            app:layout_constraintStart_toStartOf="@id/left_guidline"
+            app:layout_constraintTop_toBottomOf="@id/TV_review" />
+
+        <TextView
+            android:id="@+id/TV_movie_announce"
+            android:layout_width="0dp"
+            android:layout_height="wrap_content"
+            android:layout_marginTop="4dp"
+            android:lineHeight="20dp"
+            android:text="@string/announce"
+            android:textColor="@color/white"
+            app:layout_constraintEnd_toEndOf="@id/right_guidline"
+            app:layout_constraintStart_toStartOf="@id/left_guidline"
+            app:layout_constraintTop_toBottomOf="@id/TV_movie_story_line" />
+
+        <TextView
+            android:id="@+id/TV_movie_cast"
+            android:layout_width="0dp"
+            android:layout_height="wrap_content"
+            android:layout_marginTop="24dp"
+            android:text="@string/cast"
+            android:textColor="@color/white"
+            android:textStyle="bold"
+            app:layout_constraintEnd_toEndOf="@id/right_guidline"
+            app:layout_constraintStart_toStartOf="@id/left_guidline"
+            app:layout_constraintTop_toBottomOf="@id/TV_movie_announce" />
+
+        <ImageView
+            android:id="@+id/IV_cast_image_1"
+            android:layout_width="0dp"
+            android:layout_height="wrap_content"
+            android:layout_marginTop="7dp"
+            android:adjustViewBounds="true"
+            android:src="@drawable/image_1"
+            app:layout_constraintEnd_toStartOf="@+id/IV_cast_image_2"
+            app:layout_constraintHorizontal_chainStyle="spread"
+            app:layout_constraintStart_toStartOf="@id/left_guidline"
+            app:layout_constraintTop_toBottomOf="@id/TV_movie_cast" />
+
+        <ImageView
+            android:id="@+id/IV_cast_image_2"
+            android:layout_width="0dp"
+            android:layout_height="wrap_content"
+            android:layout_marginStart="8dp"
+            android:adjustViewBounds="true"
+            android:scaleType="centerCrop"
+            android:src="@drawable/image_2"
+            app:layout_constraintEnd_toStartOf="@+id/IV_cast_image_3"
+            app:layout_constraintStart_toEndOf="@+id/IV_cast_image_1"
+            app:layout_constraintTop_toTopOf="@id/IV_cast_image_1" />
+
+        <ImageView
+            android:id="@+id/IV_cast_image_3"
+            android:layout_width="0dp"
+            android:layout_height="wrap_content"
+            android:layout_marginStart="8dp"
+            android:adjustViewBounds="true"
+            android:scaleType="centerCrop"
+            android:src="@drawable/image_3"
+            app:layout_constraintEnd_toStartOf="@+id/IV_cast_image_4"
+            app:layout_constraintStart_toEndOf="@+id/IV_cast_image_2"
+            app:layout_constraintTop_toTopOf="@id/IV_cast_image_2" />
```

Comment:
please relate
app:layout_constraintTop_toTopOf to first image

same as it done for stars